OPHD/UI/TextRender.cpp

```diff
@@ -19,3 +19,15 @@ void drawLabelAndValue(NAS2D::Point<int> position, const std::string& title, con
 	position.x() += FONT_BOLD->width(title);
 	renderer.drawText(*FONT, text, position, NAS2D::Color::White);
 }
+
+void drawLabelAndValueLeftJustify(NAS2D::Point<int> position, const std::string& title, const std::string& text, int labelWidth, NAS2D::Color color) 
```

Comment:
This method name kind of made me go "huh?". Though after looking at what it does, I can't think of anything better. Names are hard.